### DISCUSSION TITLE
World Setting을 Horizontal로 설정할 때 화면 끊김 현상이 발생합니다.

### DISCUSSION THREAD

**Discussion by eunhye0323** [naverz/zepeto-studio-kor#276] [5. Q&A]
안녕하세요. 
World Setting을 Horizontal로 설정할 때 화면 끊김 현상이 발생하여 해결방안을 여쭙고자 질문드립니다.

Vertical로 설정하는 경우 문제없이 잘 실행되는데, Horizontal로 설정하는 경우 플레이에 지장이 있을 정도로 화면이 끊기는 현상이 발생합니다. 파일 용량이 약 13GB인데 용량으로 인해 문제가 발생하는 것인지, 아니면 특별히 문제점이 있는 것인지 궁금합니다. 하단에 World Setting 에 관한 이미지와 화면 끊김 영상을 첨부하였습니다. 

감사합니다. 

![World Setting(Horizontal)](https://user-images.githubusercontent.com/52607507/151153338-5e07c1ea-b7f8-4eb7-b53c-1cac4b947315.png)

https://user-images.githubusercontent.com/52607507/151153712-a30c1a3a-8bad-48d4-aab2-4a4a58429556.mp4

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 올려주신 영상이 보이지 않네요. 🥲
최신 월드에서도 같은 현상이 나타나는지 확인 부탁드리며, 
제페토 월드는 최대 500Mb 용량을 지원하고 있으니 용량을 줄이는 것도 큰 도움이 될 것입니다.
감사합니다.
